bridge/client/app/_views/ktb-project-view/ktb-project-view-common.module.ts

```diff
@@ -23,6 +24,7 @@ import { DtOverlayModule } from '@dynatrace/barista-components/overlay';
     DtMenuModule,
     DtOverlayModule,
     FlexModule,
+    MatDialogModule,
```

Comment:
Do we even need this module?

Reply:
I'll try to remove it. at some point I had a compile error, that no provider was found for MatDialog and this solved the problem. But I think it was for some unit tests and now it seems it's working.